jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupServiceImpl.java

```diff
@@ -439,17 +443,17 @@ public Group generateAndDeployGroupJvmFile(final String groupName, final String
             futures.put(jvm.getJvmName(), future);
         });
 
-        checkResponsesForErrorStatus(futures);
+        checkJvmGenerateAndDeployFutureErrorStatus(futures);
         return group;
     }
 
-    private void checkResponsesForErrorStatus(final Map<String, Future<Response>> futureMap) {
+    private void checkJvmGenerateAndDeployFutureErrorStatus(final Map<String, Future<Jvm>> futureMap) {
```

Comment:
Add comments/documentation on what this method does. Let's keep this as a practise going forward